src/prefect/experimental/workers/process.py

```diff
@@ -53,31 +73,48 @@ async def verify_submitted_deployment(self, deployment: Deployment):
     async def run(
         self, flow_run: FlowRun, task_status: Optional[anyio.abc.TaskStatus] = None
     ):
-        command = [sys.executable, "-m", "prefect.engine"]
+        deployment = await self._client.read_deployment(flow_run.deployment_id)
```

Comment:
Extract this to method and pass in fully populated configuration as argument to Run